src/main/scala/com/gu/acquisition/model/ReferrerAcquisitionData.scala

```diff
@@ -18,7 +18,11 @@ case class ReferrerAcquisitionData(
     source: Option[AcquisitionSource],
```

Comment:
theres a bit of an existing indentation issue going on here, not sure if you want to sort out at the same time

Reply:
i.e. 4 space change it to 2 space